website/static/js/filepage/index.js

```diff
@@ -17,6 +18,65 @@ var Panel = utils.Panel;
 
 var EDITORS = {'text': FileEditor};
 
+var MakeClient = function(element) {
```

Comment:
This is unnecessary--as it's written, `MakeClient` is exactly equivalent to `makeClient`.